integration/samples/core/specs/es5-api.ts

```diff
@@ -36,7 +36,7 @@ describe(`@sample/core`, () => {
     });
 
     it(`should import TS helpers from 'tslib'`, () => {
-      expect(BUNDLE).to.contain(`import { __spread } from 'tslib'`);
+      expect(BUNDLE).to.contain('tslib');
```

Comment:
Can we be more specific in this assertion? I have no idea how tslib is supposed to be used when looking at this assertion.

Reply:
What we care of is that tslib is present. We don't really care how it is used as that is up to the tsc compiler. tslib is used by the tsc compiler and it is emittes the helpers based on the code emitted. Previously tsickle did some transformations which didn't require tslib for certain code parts.

Re the above, question. in most cases tslib was already being included and should have for example when using the spread operator, extends etc... It was not because decorator metadata was transformed by tsickle.

tslib as no effect on lib authors and consumers.